crates/cast/bin/cmd/run.rs

```diff
@@ -52,6 +52,14 @@ pub struct RunArgs {
     #[arg(long, short)]
     verbose: bool,
 
+    /// Prints the trace in JSON format.
+    #[arg(long, short)]
+    json: bool,
+
+    /// Prints the state changes, only works with the `json` flag.
+    #[arg(long)]
+    with_state_changes: bool,
```

Comment:
`--json` should just toggle the output mode

If desired it should also be added to https://github.com/paradigmxyz/revm-inspectors so the regular output mode has optional support for it as well

Reply:
Totally agree. I'm happy to submit a separate PR to the `revm-inspectors` repo to add support for this.